Formula/ssh-audit.rb

```diff
@@ -8,7 +8,7 @@ class SshAudit < Formula
 
   bottle :unneeded
 
-  depends_on "python@2"
+  depends_on "python"
```

Comment:
```suggestion
```

Reply:
should work with system python. Unless you *really* want to be using it with py3...